05_stringer/joel00wood/main_test.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"testing"
+
+	output "github.com/joel00wood/test-helpers/capture"
+)
+
+func TestMain(t *testing.T) {
+	expected := "127.0.0.1\n"
+	actual := output.CaptureOutput(func() { main() })
+	if expected != actual {
```

Comment:
please use testify instead of raw comparisions.
Sorry I missed this during 1st review.
Rest is fine!